apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/RadioGroup.stories.mdx

```diff
@@ -0,0 +1,198 @@
+
+const useIconOptionStyles = makeStyles({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    ...shorthands.gap('5px'),
+  },
+});
+
+const useLabelStyles = makeStyles({
+  root: {
+    display: 'flex',
+    ...shorthands.gap('5px'),
+  },
+});
+
+const useGreenBorderOptionStyles = makeStyles({
+  root: {
+    ...shorthands.border('1px', 'solid', 'green'),
+  },
+});
+
+const RadioGroupCustomOptionExample = () => {
+  const labelId = useId('label-');
+  const iconOptionStyles = useIconOptionStyles();
+  const labelStyles = useLabelStyles();
+  const greenBorderOptionStyles = useGreenBorderOptionStyles();
+
+  return (
+    <>
+      <Label id={labelId} required>
+        Pick One
+      </Label>
+      <RadioGroup aria-labelledby={labelId} defaultValue="B">
+        <div className={iconOptionStyles.root}>
+          <Radio
+            value="A"
+            label={{
+              className: labelStyles.root,
+              children: (
+                <>
+                  A <AnimalCat24Regular />
+                </>
+              ),
+            }}
+          />
+        </div>
+        <Radio value="B" label="B" className={greenBorderOptionStyles.root} />
+        <Radio value="C" label="C" disabled />
+        <Radio value="D" label="D" />
+      </RadioGroup>
+    </>
+  );
+};
+```
+
+## Prop Mapping
+
+This table maps v8 `ChoiceGroup` props to the v9 `RadioGroup` equivalent.
+
+| v8                   | v9                    | Notes                                                                |
+| -------------------- | --------------------- | -------------------------------------------------------------------- |
+| `componentRef`       | `ref`                 | v9 provides access to the underlyig DOM node, not IChoiceGroup       |
+| `options`            | `children`            | v9 uses React `children` rather than data props                      |
+| `defaultSelectedKey` | `defaultValue`        | Mutually exclusive with `value`                                      |
+| `selectedKey`        | `value`               | Mutually exclusive with `defaultValue`                               |
+| `onChange`           | `onChange`            | The Typescript types have changed in v9                              |
+| `label`              | Use `Label` component | Be sure to associate `Label` with `RadioGroup` via `aria-labelledby` |
+| `theme`              | n/a                   | Use `FluentProvider` to customize themes                             |
+| `styles`             | `className`           |                                                                      |
+| `ariaLabelledBy`     | `aria-labelledby`     | In v9 this is the intrinsic HTML prop                                |
+
+This table maps v8 `IChoiceGroupOption` props to the v9 `Radio` equivalent.
+
+| v8                 | v9           | Notes                                                                                |
```

Comment:
This table is missing native HTML props like `required`, `checked`, and `name`. It's also missing some other props like `focused`, `hasImage`, etc. Were these left intentionally, or referenced somewhere else?